x/marker/types/msg_test.go

```diff
@@ -689,3 +689,94 @@ func TestMsgSetAccountDataRequestGetSigners(t *testing.T) {
 		require.PanicsWithError(t, "decoding bech32 failed: invalid separator index -1", testFunc, "GetSigners")
 	})
 }
+
+func TestMsgUpdateSendDenyListRequest(t *testing.T) {
+	addr := sdk.AccAddress("addr________________").String()
+	denom := "somedenom"
+	addAddr := sdk.AccAddress("addAddr________________").String()
+	removeAddr := sdk.AccAddress("removeAddr________________").String()
+
+	tests := []struct {
+		name          string
+		msg           MsgUpdateSendDenyListRequest
+		expectedError string
```

Comment:
Nit: I usually try to keep these field names short so that, when the values are aligned, they're all further left.

When testing something that only returns an error (or any other single value), I usually just call the field `exp`. Some times I'll call it `expErr` though.

No need to change anything. I just wanted to share. Feel free to just resolve this comment.

Reply:
I personally like the `expErr` instead of `exp` if we are going to shorten them.  I keep reading `exp` as `expression` in my head.  Since, that is what was used as at a previous job...I will get over that though.